.sphinx/_static/furo_colors.css

```diff
@@ -85,3 +85,16 @@ body {
         }
     }
 }
+
+/* suppress link underlines outside on-hover */
+a {
+    text-decoration: none;
+}
+
+a:hover {
+    text-decoration: underline;
+}
+
+a:visited:hover {
+    text-decoration-color: var(--color-link-visited);
```

Comment:
Imo it would be cleaner to put that in `furo_colors.css`, since this is where the other underline colours are defined (or can be defined). Did you have a reason why you prefer to set it here?